specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json

```diff
@@ -0,0 +1,45 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Resource Management events published to Azure Event Grid via Azure Resource Notifications",
```

Comment:
this should be pointing to health resources.

Reply:
Same for description.

Reply:
done